ipalib/constants.py

```diff
@@ -313,13 +313,7 @@
 # Use cache path
 USER_CACHE_PATH = (
     os.environ.get('XDG_CACHE_HOME') or
-    os.path.join(
-        os.environ.get(
-            'HOME',
-            os.path.expanduser('~')
-        ),
-        '.cache'
-    )
+    os.path.expanduser('~/.cache')
```

Comment:
Just one question. In config.py you test that os.path.expanduser('~') doesn't return ~. In the USER_CACHE_PATH case the return value could be the string '~/.cache'. Is that a problem? Should it instead fall back to trying $HOME?